CONTRIBUTING.md

```diff
@@ -1,5 +1,18 @@
 # Contributing to the Cardano Ledger
 
+## Roles and responsibilities
+
+The
+[@cardano-ledger](https://github.com/orgs/input-output-hk/teams/cardano-ledger)
+group is responsible for
+helping with reviewing and merging pull requests, adjudicating technical (or other) disputes,
+releasing the ledger packages on [CHaP](https://github.com/input-output-hk/cardano-haskell-packages).
+
+@hamishmack can help with issues regarding this repository's continuous integration and nix infrastructure.
+
+**For security related issues** please consult the security file in the
```

Comment:
I would recommend actually making a `SECURITY.md` that just says `See <link>`, because then github will notice that you have it and say you have a security policy. Same for the CoC. I'll clarify that in the handbook.

Reply:
Oh I mean, you already did that for the CoC.

Reply:
I did that as well, you'll the `SECURITY.md` file added in this PR. This was just another place I thought I would make mention of it.